runtime-light/component/component.h

```diff
@@ -63,6 +64,10 @@ struct ComponentState {
 
   task_t<void> run_component_epilogue() noexcept;
 
+  ComponentKind component_kind() const noexcept {
```

Comment:
Is inline will be generated here automatically?

Reply:
Yep. See https://en.cppreference.com/w/cpp/language/inline